curriculum/challenges/english/25-front-end-development/workshop-reusable-mega-navbar/6745d9df333efe2543a1f457.md

```diff
@@ -0,0 +1,167 @@
+---
+id: 6745d9df333efe2543a1f457
+title: Step 8
+challengeType: 0
+dashedName: step-8
+---
+
+# --description--
+
+Still inside the third `li` element, create a `ul` element for the popup. Give it a `className` of `sub-menu` and an `aria-label` of `submenu`.
```

Comment:
```suggestion
Still inside the third `li` element, create a `ul` element for the popup. Give it a `className` of `sub-menu`.
```

This list doesn't need to be named. It will be clear to screen reader users that it is the submenu for Apps because it follows the Apps button directly in the DOM.

Reply:
Thank you